### PR TITLE
dotCMS/core#19542 Fix the logout on SESSION_DESTROYED event from websocket

### PR DESCRIPTION
**Pull request by fmontes**
…

### CODE REVIEW

**Review by hmoreras — APPROVED** [review]

File: projects/dotcms-js/src/lib/core/login.service.ts [hunk 1]
```diff
@@ -344,16 +315,12 @@ export class LoginService {
...
-    // private loadLoginAsUsersList(includeNUsers: boolean, filter: string): Observable<any> {
-    //     return this.coreWebService
-    //         .requestView({
-    //             url: `${this.urls
-    //                 .loginAsUserList}?includeUsersCount=${includeNUsers}&filter=${filter}`
-    //         })
-    //         .pluck('entity');
-    // }
+    private logOutUser(): void {
+        window.location.href = LOGOUT_URL;
```
Comment (freddyucv) [inline projects/dotcms-js/src/lib/core/login.service.ts:322]: is not need to do this request anymore?

https://github.com/dotCMS/core-web/pull/1510/files#diff-cd0ba6e2364eca90ea6a4f00fe17b2869f58b1ab31a8abbd2a16b48674914ea4L251

File: projects/dotcms-js/src/lib/core/login.service.ts [hunk 2]
```diff
@@ -18,6 +17,8 @@ export interface DotLoginParams {
     backEndLogin: boolean;
 }
 
+export const LOGOUT_URL = '/dotAdmin/logout';
```
Comment (freddyucv) [inline projects/dotcms-js/src/lib/core/login.service.ts:20]: is it possible that the user would like to override this URL?

Comment (fmontes) [reply projects/dotcms-js/src/lib/core/login.service.ts:20]: Not at the moment is a SAML thing.

File: projects/dotcms-js/src/lib/core/login.service.ts (hunk 1 above)
Comment (fmontes) [reply projects/dotcms-js/src/lib/core/login.service.ts:322]: Not... SAML.

**Review by freddyucv — APPROVED** [review]

**Pull request merged**